Let a minion contain an avahi reflector if desired (#532)
moio/sumaform: modules/openstack/README.md

## Accessing VMs

All machines come with user `root` with password `linux`. They are also accessible via your SSH public key (by default `~/.ssh/id_rsa.pub`) if you have one.

By default, the machines use Avahi (mDNS), and are configured on the `.tf.local` domain. Thus if your host is on the same network segment of the virtual machines you can simply use:
```
ssh root@susemanager-suma31pg.tf.local
```

If you use Avahi and are on another network segment, you can only connect using an IP address, because mDNS packets do not cross network boundaries unless using reflectors.

> To determine the IP addresses, you can look in the OpenStack admin Web UI for the floating IPs.

Reviewer: ```suggestion
IP addresses, called floating IPs, are outputted by `terraform apply` and can also be seen in the OpenStack admin Web UI.
```

Author: OK (but I'm not sure "outputted" exists. I think the irregular past of "put" is "put" :tongue: .

Reviewer: Whoops :man_facepalming:

If you want to use a different SSH key, or mDNS does not work out of the box, or if you don't want to use mDNS, please check the README_ADVANCED.md and TROUBLESHOOTING.md files.

Web access is on standard ports, so `firefox <FLOATING_IP_ADDRESS>` will work as expected. SUSE Manager default user is `admin` with password `admin`.

> Finally, you should be able to use the Openstack web console to access your VM.

Reviewer: ```suggestion
Finally, you can use the OpenStack web console to access your VM.
```

Author: Yup.
